requirements.txt

```diff
@@ -1,6 +1,6 @@
 Cython>=0.28,<1.0; python_version == '3.7'
 PyStemmer==1.3.0
-bblfsh==2.12.7
+bblfsh==2.9.13
```

Comment:
@zurk I think that setting the version to 2.9.13 will break the downstream packages which require a newer version... Thoughts?

Reply:
The best option I see is to ask data retrieval team to update bblfsh requirement and publish the new package. The problem is `sourced-jgit-spark-connector` is deprecated...

Reply:
Yes, it is not going to be updated, also there may be a backward incompatibility. Besides, we will soon have to switch to v3. I think that we should accept this change and work around it in dependent projects.

Reply:
agree.